dash.el

```diff
@@ -1048,6 +1048,21 @@ This function can be thought of as a generalization of
   "Return a list of ((-filter PRED LIST) (-remove PRED LIST)), in one pass through the list."
   (--separate (funcall pred it) list))
 
+(defun -separate-multi (preds list)
+  "Return a list of ((-filter PRED1 LIST) (-filter PRED2 LIST) ... REST), in one
+  pass through the list. PREDS is a list of functions and REST are elements of
+  LIST that are unmatched by any PREDS."
+  (setq preds (mapcar #'list (-snoc preds (lambda (_) t))))
+  (dolist (elem list)
+    (--first (when (funcall (car it) elem)
+               (push elem (cdr it)))
+             preds))
+  (--map (nreverse (cdr it)) preds))
+
+(defmacro --separate-multi (forms list)
+  "Anaphoric form of -separate-multi."
```

Comment:
Please quote symbol references as ``` `-separate-multi' ``` so that the help system can hyperlink them, as per the aforementioned conventions.